app/src/main/res/layout/layout3.xml

```diff
@@ -93,7 +93,7 @@
                 android:text="Name"
                 app:layout_constraintStart_toStartOf="parent"
                 tools:layout_editor_absoluteY="100dp"
-                tools:text="Medicine Name" />
+                tools:text="Medication Name" />
```

Comment:
Changed Medicine lines to Medication. Medication was used more frequently in the app then medicine was, so making this change made the UI more consistent.